web_src/css/features/projects.css

```diff
@@ -80,12 +80,15 @@
 .board-card {
   margin: 4px 2px !important;
   border-radius: 5px !important;
-  cursor: move;
   width: calc(100% - 4px) !important;
   padding: 0.5rem !important;
   min-height: auto !important;
 }
 
+.board-card.sortable {
+  cursor: move;
+}
+
```

Comment:
Oops, I think this could be improved:

1. Revert `class="card board-card {{if $.CanWriteProjects}}sortable{{end}}"`
2. Change this to: `.board.sortable .board-card`

Reply:
Done.